### DISCUSSION TITLE
.NET Windows Forms C#

### DISCUSSION THREAD

**Discussion by bananaNuggets75** [microsoft/vscode-discussions#1181] [Extension Development QnA]
is winforms still not available on VSCode mac? Since the Visual Studio for mac will be expiring in August, I am migrating to VSCode, but as I have installed every extensions and packages related to .NET Windows Form (C#) I still cannot find or run winforms on vscode. do you suggest another IDE or extensions perhaps

**Comment by isidorn** [comment]
This forum is for vscode extension authors. This is off-topic, and I will close this.
Please ask this on stack overflow
